### DISCUSSION TITLE
windows10下通过webdav挂载百度网盘，上传文件失败

### DISCUSSION THREAD

**Discussion by whjcd1998** [AlistGo/alist#1928] [Q&A] [answered]
在win10的资源管理器中向挂载的文件夹中复制文件失败，一直卡在该界面：
![image](https://user-images.githubusercontent.com/57035180/194712212-29a75ffc-16d7-43f1-8f70-2b242a7e2d40.png)
命令行报错如下：
![image](https://user-images.githubusercontent.com/57035180/194711706-619c4637-01c3-4dc9-a7b2-b4552faf6d5b.png)
可正常下载文件。
在alist的网页端可正常下载及上传。
请问可能是哪里出了问题？

**Comment by youtehub** [comment, marked as answer]
有两种方案可以挂载到本地硬盘，raidrive，和rclone，raidrive去网上搜索教程，非常好使用，不过只能使用旧版本才能实现上传，下载，最新的版本只能实现播放电影，下载文件，即使开了**会员**也未必可以实现上传，最新版有bug。
rclone方案的话，我目前实现了上传，下载，
```
rclone mount baidu:/ P: --cache-dir F:\RcloneCache --vfs-cache-mode writes --header "Referer:https://pan.baidu.com/"  --header "User-Agent:pan.baidu.com" 
```
![image](https://user-images.githubusercontent.com/18629921/194759892-11e04fdb-dfd6-47a8-884e-d24e19efcace.png)

这个配置时我探索了好几天才找到的方案，当然你自己也可以添加其他参数，关键就在  --header 这个参数上，目前测试可以实现上传下载了

**Reply by im2058** [reply]
能方便再测试下么，前几天还可以，现在又不行了

**Comment by youtehub** [comment]
博客在这里，附上了正在使用的脚本，拿取自己调整一下
博客地址：[https://blog.csdn.net/csdn_life18/article/details/127234864]( https://blog.csdn.net/csdn_life18/article/details/127234864)

**Reply by im2058** [reply]
![image](https://user-images.githubusercontent.com/30195987/195970291-5d44ee4e-7314-41f1-a3f7-e4d86b9c73cd.png)
